ddtrace/internal/core/__init__.py

```diff
@@ -373,5 +372,5 @@ def get_span() -> Optional["Span"]:
 def get_root_span() -> Optional["Span"]:
```

Comment:
I'd recommend deprecating this and any other functions here that use Spans. Hopefully we'll be able to completely excise Spans from `core` before too long.